docs/releasing/publishing-a-pre-release.md

```diff
@@ -0,0 +1,135 @@
+# Before you publish a pre-release of GOV.UK Frontend
+
+Decide the kind of pre-release you're publishing. All pre-releases are public, but we distinguish:
+
+- `internal` pre-releases for internal use
+- `beta` pre-releases aimed at external users
+
+We'll use the kind of pre-release as [the pre-release identifier](https://docs.npmjs.com/cli/v8/commands/npm-version#preid) in the package version.
+
+## If you're publishing a beta pre-release
+
+Besides publishing the code itself, beta releases will likely involve
+documentation updates linked to the code, as well as communications, similarly to an actual release.
+
+Review the docs for [what to do before publishing a release](/docs/releasing/before-publishing-a-release.md) to assess which steps you need to follow for your specific pre-release and ensure you are prepared to publish.
```

Comment:
Again, it'd be great if we could just be sure that for beta releases, we follow a normal release process rather than leaving it for individuals to assess. But no worries if that's not possible right now.

Reply:
@domoscargin I think that's something we'll only be able to pin down after a couple of betas. We can clarify once we've got a few under our belt, for different kinds of releases and see what's common and what's flexible from one to another.